src/porepy/numerics/fv/fvutils.py

```diff
@@ -671,94 +671,98 @@ def __init__(self, subcell_topology, bound, nd):
         fno = subcell_topology.fno_unique
         num_subfno = subcell_topology.num_subfno_unique
 
+        self.fno = fno
```

Comment:
This whole class (ExcludeBoundaries) is a monster! I did manage to clean up the __init__, however it still have 14 (!) methods that basically does the same. We are apparently doing something wrong with the boundary conditions

Reply:
I agree with both. It started out as a simple concept, but then it snowballed. Feel free to create an issue on it.